src/Umbraco.Cms.Api.Management/Controllers/FileSystemManagementControllerBase.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Net;
+
+namespace Umbraco.Cms.Api.Management.Controllers;
+
+public abstract class FileSystemManagementControllerBase : ManagementApiControllerBase
+{
+    protected string DecodePath(string path)
```

Comment:
I wonder, if this should be a custom modelbinder instead, to keep the controllers more clean

Reply:
It would be awesome if we could. Unfortunately, .NET Core modelbinding of path parameters doesn't seem to play well with Swagger UI 😢